### PR TITLE
Document `AStar.get_point_path()` not being thread-safe

### PR DESCRIPTION
**Pull request by Calinou**
See https://github.com/godotengine/godot/issues/19311.

### CODE REVIEW

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.3.

File: doc/classes/AStar.xml [hunk 1]
```diff
@@ -289,6 +289,7 @@
 			</argument>
 			<description>
 				Returns an array with the points that are in the path found by AStar between the given points. The array is ordered from the starting point to the ending point of the path.
+				[b]Note:[/b] This method is not thread-safe. If called from a [Thread], it will return an empty [PackedVector2Array] and will print an error message.
```
Comment (akien-mga) [inline doc/classes/AStar.xml:292]: This should have been `PackedVector3Array` (you can maybe include the fix in another doc PR, e.g. #47772 which needs an amend too).

Comment (Calinou) [reply doc/classes/AStar.xml:292]: I amended https://github.com/godotengine/godot/pull/47772 to contain this fix.

**Pull request merged**